### PR TITLE
[ADD] helpdesk_stock

### PR DESCRIPTION
**Pull request by osimallen**
#11

### CODE REVIEW

**Comment by osimallen** [comment]
I wasn't sure how to credit Brian on this module. I reused much of the code from his fieldservice_stock module. For now, I have just added his name under the contributors.

**Comment by max3903** [comment]
@brian10048 Any issue with your code from fieldservice_stock being copied and relicensed (LGPL-3) here?

**Comment by brian10048** [comment]
@max3903 Not a problem at all

**Comment by max3903** [comment]
@brian10048 Thank you very much.

File: helpdesk_stock/README.rst [hunk 1]
```diff
@@ -0,0 +1,39 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+===========================
+Helpdesk - Stock
+===========================
```
Comment (max3903) [inline helpdesk_stock/README.rst]: The `=` lines should match the length of the title.

File: helpdesk_stock/README.rst [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+=====
+
+Create a helpdesk ticket. On the form, you will see an inventory tab.
+On the inventory tab, the user can add items that need to be shipped 
+to complete the ticket. 
+
+In the inventory app, under operations, a list of helpdesk tickets with
+associated stock moves are listed if you select the Helpdesk Tickets 
+menu option. When a shipment occurs, the status change will be reflected
+on the inventory tab of the helpdesk ticket.
```
Comment (max3903) [inline helpdesk_stock/README.rst]: @osimallen Can you use the content from #11?

**Review by max3903 — CHANGES_REQUESTED** [review]
@osimallen Minor changes. :+1:

**Comment by max3903** [comment]
@osimallen Can you also add this module to the exclude list in `travis.yml` as we can't test it?

File: helpdesk_stock/README.rst (hunk 1 above)
Comment (osimallen) [reply helpdesk_stock/README.rst]: Completed in new commit

File: helpdesk_stock/README.rst (hunk 2 above)
Comment (osimallen) [reply helpdesk_stock/README.rst]: Completed in new commit

**Review by max3903 — APPROVED** [review]

**Pull request merged**